tests/cupy_tests/lib_tests/test_polynomial.py

```diff
@@ -22,7 +32,7 @@ def test_poly1d_numpy_array(self, xp, dtype):
         with cupyx.allow_synchronize(False):
             out = xp.poly1d(a, variable=self.variable)
         assert out.variable == (self.variable or 'x')
-        return out.coeffs
```

Comment:
This fix is unrelated to this PR. Please split into another PR if you want to work on the refactoring.